libdebug/debugger/debugger.py

```diff
@@ -72,7 +72,8 @@ def terminate(self: Debugger) -> None:
 
     def cont(self: Debugger) -> None:
```

Comment:
The idea isn't the worst, but it introduces an inconsistency between the continue function and the other functions. Typically, if no thread is specified, the action is applied to threads[0]. However, this isn't the case for continue. We can document this exception, but we need to carefully consider its implications.

You were waiting for this comment, I know...